### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Added Todos
Saved Todos to local storage
Users can edit, delete, and tick off their todos from the list

image

image

image

image

### CODE REVIEW

File: src/components/Todo.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    if (e.key === "Enter") {
+      const id = Math.random();
+      setTodos((updateTodos) => [
+        ...updateTodos,
+        { id, text, completed: false },
+      ]);
+      setText("");
+    }
+  };
+  console.log(todos.length);
```
Comment (aniketsingh13) [inline src/components/Todo.jsx]: remove console

**Review by aniketsingh13 — APPROVED** [review]
ui looks good 
code is easy to read
good to merge

**Review by sashaboi — APPROVED** [review]
Looks amazing ! 

A suggestion , you could put a little bit of opacity or darker wallpaper for better visibility of texts. 
Also could put loaders where api is in loading state like in weather.

File: src/components/Todo.jsx (hunk 1 above)
Comment (ranitasahaa1230) [reply src/components/Todo.jsx]: Will fix this

**Comment by ranitasahaa1230** [comment]
> Looks amazing !
> A suggestion , you could put a little bit of opacity or darker wallpaper for better visibility of texts. Also could put loaders where api is in loading state like in weather.

Sure will work on it

File: public/index.html [hunk 2]
```diff
@@ -9,7 +9,11 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <meta name="author" content="Ranita Saha" />
+    <meta
+      name="description"
+      content="Moments Of The Day replaces new tab screen with your today's Main Focus, Weather, Time and Quotes."
+    />
```
Comment (Shailseen) [inline public/index.html:16]: Liked the use of meta tags for SEO.

File: src/components/Todo.jsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      );
+      setTodos(updateTodo);
+      setEditID(0);
+      setText("");
+      return;
+    }
+  };
+  const handleTodo = (e) => {
+    if (e.key === "Enter") {
+      const id = Math.random();
```
Comment (Shailseen) [inline src/components/Todo.jsx:29]: You can use something like uuid for id's

**Review by Shailseen — COMMENTED** [review]
Liked the UI and structure of project. Good work

**Pull request merged**